docs/en/loading/load_concept/strict_mode.md

```diff
@@ -61,9 +61,9 @@ If strict mode is enabled, StarRocks loads only the rows that hold `\N` or `1` a
 
 ## Set strict mode
 
-If you run a [Stream Load](../../sql-reference/sql-statements/loading_unloading/STREAM_LOAD.md), [Broker Load](../../sql-reference/sql-statements/loading_unloading/BROKER_LOAD.md), [Routine Load](../../sql-reference/sql-statements/loading_unloading/routine_load/CREATE_ROUTINE_LOAD.md), or [Spark Load](../../sql-reference/sql-statements/loading_unloading/SPARK_LOAD.md) job to load data, use the `strict_mode` parameter to set strict mode for the load job. Valid values are `true` and `false`. The default value is `false`. The value `true` enables strict mode, and the value `false` disables strict mode.
+You can use the `strict_mode` parameter to set strict mode for the load job. Valid values are `true` and `false`. The default value is `false`. The value `true` enables strict mode, and the value `false` disables strict mode. Note the `strict_mode` parameter is supported for INSERT from v3.4.0.
 
-If you execute [INSERT](../../sql-reference/sql-statements/loading_unloading/INSERT.md) to load data, use the `enable_insert_strict` session variable to set strict mode. Valid values are `true` and `false`. The default value is `true`. The value `true` enables strict mode, and the value `false` disables strict mode.
+You can also use the `enable_insert_strict` session variable to set strict mode. Valid values are `true` and `false`. The default value is `true`. The value `true` enables strict mode, and the value `false` disables strict mode.
```

Comment:
give a notice: It has a little behavior change ...

Reply:
added in system variable doc